src/App.js

```diff
@@ -0,0 +1,106 @@
+import React, {useState} from 'react';
+import Screen from './components/Screen/Screen'
+import Button from './components/Button/Button'
+import './App.css';
+
+function App() {
+  const [screenText, setScreen] = useState('0')
+  function handleClick(text) {
+    if (text === '=') {
+      const screenSplit = screenText.split(' ')
+      if (screenSplit.length <= 2) {
+        setScreen('error')
+      } else if (screenSplit.length > 3) {
+        setScreen('error')
+      } else if (screenSplit[1] === '÷') {
+        setScreen((screenSplit[0] / screenSplit[2]).toString())
+      } else if (screenSplit[1] === '×') {
+        setScreen((screenSplit[0] * screenSplit[2]).toString())
+      } else if (screenSplit[1] === '-') {
+        setScreen((parseFloat(screenSplit[0]) - parseFloat(screenSplit[2])).toString())
+      } else if (screenSplit[1] === '+') {
+        setScreen((parseFloat(screenSplit[0]) + parseFloat(screenSplit[2])).toString())
+      } else {
+        setScreen('error')
+      }
+    } else if (screenText === '0' && text !== '⇐') {
+      setScreen(text)
+    } else if (text === '⇐' || text === 'clr') {
+      if (screenText.length === 1 || screenText === 'error' || text === 'clr') {
+        setScreen('0')
+      } else {
+        setScreen(screenText.substring(0, screenText.length - 1))
+      }
+    } else if (text === '÷' || text === '×' || text === '-' || text === '+' || text === 'y²' || text === '1/x' || text === '√') {
+      switch (text) {
+        case '÷':
+          setScreen(screenText + ' ÷ ')
+        break;
+        case '×':
+          setScreen(screenText + ' × ')
+        break;
+        case '-':
+          setScreen(screenText + ' - ')
+        break;
+        case '+':
+          setScreen(screenText + ' + ')
+        break;
```

Comment:
In the `switch` statement you could do something like `return setScreen(screenText + ' - ')` instead calling the function then `break`